docat/tests/test_search.py

```diff
@@ -0,0 +1,302 @@
+import io
```

Comment:
Nice tests!

Search by file content is not tested as far as I can see, or i missed it :)

Reply:
Yeah, I thought I wrote those tests but maybe not :)